tests/SonataDoctrineORMAdminBundleTest.php

```diff
@@ -25,17 +26,21 @@ class SonataDoctrineORMAdminBundleTest extends TestCase
 {
     public function testBuild()
     {
-        $containerBuilder = $this->createMock(ContainerBuilder::class);
+        $containerBuilder = $this->prophesize(ContainerBuilder::class);
 
-        $containerBuilder->expects($this->exactly(3))
-            ->method('addCompilerPass')
-            ->withConsecutive(
-                [$this->isInstanceOf(AddGuesserCompilerPass::class)],
-                [$this->isInstanceOf(AddTemplatesCompilerPass::class)],
-                [$this->isInstanceOf(AddAuditEntityCompilerPass::class)]
-            );
+        $containerBuilder
+            ->addCompilerPass(Argument::type(AddGuesserCompilerPass::class))
+            ->shouldBeCalledTimes(1);
```

Comment:
All this can be simplified to shouldBeCalled().
And could be done in one line probably (have to check chars)

Reply:
Well, I can do it in one line and check for `CompilerPassInterface` instead of specific class.

Reply:
That's another option. There are certain things that can be done with phpunit mocks that are not direct translated to how you do it with prophecy, but there is always a way in which you get a cleaner code. atleast IMO.